include/zephyr/drivers/can.h

```diff
@@ -473,6 +477,12 @@ STATS_SECT_ENTRY32(crc_error)
 STATS_SECT_ENTRY32(form_error)
 STATS_SECT_ENTRY32(ack_error)
 STATS_SECT_ENTRY32(rx_overrun)
+STATS_SECT_ENTRY32(tx_warning)
+STATS_SECT_ENTRY32(rx_warning)
+STATS_SECT_ENTRY32(busoff)
+STATS_SECT_ENTRY32(frame_error)
+STATS_SECT_ENTRY32(fd_error)
+STATS_SECT_ENTRY32(passive_error)
```

Comment:
Please leave these additions out.

Reply:
I left out the update for support counter statistic